access-protected-api-obo/README.md

```diff
@@ -0,0 +1,157 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- java
+page_type: sample
+name: "Java API written in Spring Boot that both protects its own endpoints and calls Graph on-behalf-of the client."
+description: "This Java API that protects its own endpoints using JWT scope validation and calls Microsoft Graph on behalf of the client. The code in this sample is used by one or more articles on docs.microsoft.com."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-webapp-java
+---
+-->
+
+# Java Spring Boot | web API | access control (protected routes), protected web API access (Microsoft Graph)  | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This Java Spring Boot web API uses the Microsoft Authentication Library (MSAL) for Java to require authorization to access a JWT-protected route, which then calls Microsoft Graph on behalf of the client.
+
+```console
+$ curl http://localhost:8080/me -H "Authorization: Bearer {valid-access-token}"
+{
+  "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity",
+  "businessPhones": ["+1 (999) 5550001"],
+  "displayName": "Hans Gruber",

```

Comment:
Per CELA, we have to use _our_ fictitious names for organizations and people. Let's use **Contoso** for the org, and **Contoso Employee** for now.

The username I used to use is no longer usable (this is a good story, remind me on Monday to tell you about it.).

Reply:
Updated to "Contoso Employee" in both spots in this README.

Changed in https://github.com/Azure-Samples/ms-identity-docs-code-java/pull/5/commits/2738ee798b1c4ad5106dbd3642a9b6dff6075020